generator/C/include_v2.0/mavlink_helpers.h

```diff
@@ -681,11 +676,12 @@ MAVLINK_HELPER uint8_t mavlink_frame_char_buffer(mavlink_message_t* rxmsg,
                         status->parse_state = MAVLINK_PARSE_STATE_GOT_PAYLOAD;
                     }
 #ifdef MAVLINK_CHECK_MESSAGE_LENGTH
-                    if (rxmsg->len != MAVLINK_MESSAGE_LENGTH(rxmsg->msgid))
+                    if (rxmsg->len < mavlink_min_message_length(rxmsg) ||
+                        rxmsg->len > mavlink_max_message_length(rxmsg))
```

Comment:
I have a feeling one of these bounds checks is missing a >= or <=

Reply:
I don't know :thinking:.